pkg/volume/gce_pd/attacher.go

```diff
@@ -165,6 +165,9 @@ func (attacher *gcePersistentDiskAttacher) MountDevice(spec *volume.Spec, device
 	options := []string{}
 	if readOnly {
 		options = append(options, "ro")
+	} else {
+		// as per https://cloud.google.com/compute/docs/disks/add-persistent-disk#formatting
```

Comment:
We need to test for ext4 also. I don't think ext2 (nor implictly ext3 supports this mount option)?